dictionaries/CallMap.php

```diff
@@ -5437,14 +5442,14 @@
 'imageObj::pasteImage' => ['void', 'srcImg'=>'imageObj', 'transparentColorHex'=>'int', 'dstX'=>'int', 'dstY'=>'int', 'angle'=>'int'],
 'imageObj::saveImage' => ['int', 'filename'=>'string', 'oMap'=>'mapObj'],
 'imageObj::saveWebImage' => ['string'],
-'imageopenpolygon' => ['bool', 'image'=>'GdImage', 'points'=>'array', 'num_points'=>'int', 'color'=>'int'],
+'imageopenpolygon' => ['bool', 'image'=>'GdImage', 'points'=>'array{0: int, 1: int, 2: int, 3: int}', 'color'=>'int'],
```

Comment:
I think `non-empty-list<int>` might be a better fit here. Array shapes are open, meaning `[1, 2, 3, 4, "a" => "b"]` satisfies `array{0: int, 1: int, 2: int, 3: int}`. We don't have a way to guarantee that there are an even number of items in the list so the type isn't going to be perfect no matter what we use, and I don't think guaranteeing at least 4 elements is worth losing the `list` type. If we later add support for a list type with a minimum number of items we can change it then.